public/wp-content/themes/glific/src/scss/style.scss

```diff
@@ -55,6 +55,34 @@ body{
 	border: 3px solid $theme-pewter;
 }
 
+.get-started-content-box {
+    box-shadow: 6px 6px 9px -2px rgba($color: #000000, $alpha: 0.1);
+}
+
+
+.card-header .btn[aria-expanded=true] {
```

Comment:
It seems like you are overriding the whole card styling. can you please make it specific for your section?

Reply:
Created a separate SCSS file for page FAQ.